pkg/cmd/recoveryapiserver/create.go

```diff
@@ -0,0 +1,122 @@
+package recoveryapiserver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/client-go/tools/watch"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/recovery"
+)
+
+type CreateOptions struct {
+	Options
+
+	KubeApiserverImage string
+	Timeout            time.Duration
+	Wait               bool
```

Comment:
wait for what?

Reply:
for the apiserver healthz https://github.com/openshift/cluster-kube-apiserver-operator/blob/3a5f313893a373f7b3f766d3c08ef17ce6f19fed/pkg/cmd/recoveryapiserver/create.go#L115